### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by abhishektanwar**
Files to check:

- src/screens/Wishlist/index.jsx
- src/screens/Wishlist/wishlist.css

### CODE REVIEW

File: src/screens/Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.wishlist-container{
+  max-width: 1000px;
```
Comment (zoyatanweer) [inline src/screens/Wishlist/wishlist.css:2]: this can be in rem

**Review by zoyatanweer — APPROVED** [review]
code is clean
easy to understand
good to merge!

File: src/components/CardSlider/card-slider.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  height: max-content;
+  position: relative;
+}
+
+.card-slider{
+  width: 100%;
+  height: max-content;
+  display: flex;
+  white-space: nowrap;
+  /* flex-wrap:nowrap; */
```
Comment (santhoshpemmaka) [inline src/components/CardSlider/card-slider.css:22]: if possible remove the comment line.

**Comment by santhoshpemmaka** [comment]
Small suggestion: remove comment lines next PR request. Apart from that code looks good and clean 💯 . Good to merge 👍 .

**Review by santhoshpemmaka — APPROVED** [review]

**Pull request merged**